tardis/tests/integration_tests/report.py

```diff
@@ -67,14 +70,16 @@ def __init__(self, dokuwiki_details):
         else:
             self.dokuwiki_url = dokuwiki_details["url"]
 
+    def pytest_sessionstart(self, session):
```

Comment:
I *think* you don't need this as you should be able to convert suite_start_time to utc